source/process.adoc

```diff
@@ -8,9 +8,9 @@
  * SPDX-License-Identifier: EPL-2.0
 ////
 [[efsp]]
-= Eclipse Foundation Specification Process v1.1
+= Eclipse Foundation Specification Process
 
-Version 1.1. Effective March 20, 2019
+DRAFT Version 1.2. Effective June 18, 2019
```

Comment:
Maybe this "DRAFT" can be changed into a doc variable so that it can be modified as part of the final build process for the EFSP?  We do that with our MicroProfile spec documents.  During our build process, we parameterize the [Draft/Final] version of the release and that is used in our pdf and html generation of the Spec document.  Something similar for the EFSP might be nice.

Reply:
Good idea. Can I assume that you've set up those builds so that they say "DRAFT" unless specified otherwise?

Reply:
Moved to issue #18.